channels.yaml

```diff
@@ -548,14 +554,20 @@ releases:
     serverArgs: *serverArgs-v7
     agentArgs: *agentArgs-v5
     featureVersions: *featureVersions-v1
-  - version: v1.28.7+k3s1
+  - version: v1.27.12+k3s1
+    minChannelServerVersion: v2.8.0-alpha1
+    maxChannelServerVersion: v2.8.99
+    serverArgs: *serverArgs-v7
+    agentArgs: *agentArgs-v5
+    featureVersions: *featureVersions-v1
+  - version: v1.28.8+k3s1
     minChannelServerVersion: v2.8.3-alpha1
     maxChannelServerVersion: v2.8.99
-    serverArgs:
+    serverArgs: &serverArgs-v7
       <<: *serverArgs-v6
       embedded-registry:
         type: boolean
-    agentArgs:
+    agentArgs: &agentArgs-v5
```

Comment:
This can all probably pull directly `*serverArgs-v7` and `*agentArgs-v5` as they seemed to be defined in the same way